### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by jdescelliers** [PowerShell/PowerShell#15436]
Hi,

While PS is quite useful for many tasks, I find that basic things like writing/editing a file is not as easy as it would be on unix/linux that are usually bundled with `vi` or `nano`.

I understand that it might not be the right place to this feature request, so there might be a better place for this feature request.

Maybe the old `edit.exe` would do the trick?

Is there any plans to provide such tool for Windows command line?

I believe `vi` is being able to handle very large files which could be a plus.

**Comment by 237dmitry** [comment]
In fact, there are modern console text editors. Try [micro](https://github.com/zyedidia/micro/releases) or the built-in [Far Manager](https://github.com/FarGroup/FarManager/releases) editor. Both of them have syntax highlighting, including powershell scripts.

**Comment by asmith3006** [comment]
Whilst I appreciate that there are installable ones available, a native one which you can rely on when you remote into any server (as long as the version of PS is up to date) would be much better. 
 
I cannot install apps I want on a remote server without a long approval process, so a built in one would be much more useful. 

I don't even care about syntax highlighting, just being able to change some text in remote config files would be a huge help.

**Comment by rkeithhill** [comment]
I like the suggestion, but it feels more like a suggestion for Windows instead of PowerShell.

**Comment by asmith3006** [comment]
I disagree, it being universal on Win, Mac and Linux would be very nice in my opinion.
